vm/error.go

```diff
@@ -1,18 +1,95 @@
 package vm
 
-type ErrorClass struct {
-	*BaseClass
+import (
+	"fmt"
+)
+
+var (
+	// UndefinedMethodErrorClass ...
+	UndefinedMethodErrorClass *RClass
+	// ArgumentErrorClass ...
+	ArgumentErrorClass *RClass
+	// TypeErrorClass ...
+	TypeErrorClass *RClass
+)
+
+func initErrors() {
+	bc := createBaseClass("UndefinedMethodError")
```

Comment:
You can set these exception name into constants like

```go
const(
    UndefinedMethodError = "UndefinedMethodError"
)
```

So that we can use this constants in other places instead of copy and paste strings.